sycl/plugins/unified_runtime/ur/ur.hpp

```diff
@@ -42,6 +42,9 @@ const ur_device_info_t UR_EXT_DEVICE_INFO_OPENCL_C_VERSION =
 const ur_device_info_t UR_EXT_DEVICE_INFO_MAX_REGISTERS_PER_WORK_GROUP =
     (ur_device_info_t)((uint32_t)UR_DEVICE_INFO_FORCE_UINT32 - 1);
 
+const ur_device_info_t UR_EXT_DEVICE_INFO_ESIMD_SUPPORT =
+    (ur_device_info_t)((uint32_t)UR_DEVICE_INFO_FORCE_UINT32 - 2);

```

Comment:
I am extremely confident this is completely and horribly wrong, but I'm not exactly sure what to do. Do we need to add this to the unified runtime spec?